Tasks/AzureResourceManagerTemplateDeploymentV3/operations/Utils.ts

```diff
@@ -444,6 +444,22 @@ class Utils {
                 throw new Error(tl.loc("AzureCLINotFound"));
             }
         }
+        
+        if(filePathExtension === 'bicepparam'){
+            var fileName: string = filePath.split(path.sep).pop().split('.')[0];
```

Comment:
Can we use the existing method and based on the file extension  we call the method accordingly so that we can reuse the common functions

Reply:
Changes for this have been moved into existing methods.